src/containers/Tenant/Diagnostics/Describe/Describe.tsx

```diff
@@ -60,7 +88,10 @@ const Describe = ({tenant}: IDescribeProps) => {
         );
     };
 
-    if (loading && !wasLoaded) {
+    // Schema loads children paths.
+    // To prevent rendering describe without children where children are needed,
+    // schema loading state is checked here
+    if (!wasLoaded || (hasNested && schemaLoading && !schemaWasLoaded)) {
```

Comment:
why don't you check describe `loading` anymore?

Reply:
I tried to get rid of this kind of loading check.

My idea is in separating two flags, `loading` and `wasLoaded`, for different purposes. First is for loading data in a background mode, second is for the initial fetch (or fetch on path change), where loader should be shown. And wasLoaded is set only when specific argument is passed to fetch function. 

This way we could do both background fetches or fetches from different components (SchemaTree, Tenant, etc.) without loader reset.

As for the `loading`, I don't know why it is needed. Maybe we can even delete it. But I'm not sure, maybe there was some other logic, that depends on such flags